config/application.rb

```diff
@@ -31,7 +31,7 @@ class Application < Rails::Application
     # to allow any host to connect to it. In production, set it to a comma-separated list of
     # domains where the client application is known to be hosted.
     # # => KNOWN_HOSTS=clientside.knowplace.com,another.pla.ce
-    KNOWN_HOSTS = ENV.fetch('KNOWN_HOSTS') { 'http://lvh.me:5000,http://localhost:4200,http://10.10.2.119:4200' }
+    KNOWN_HOSTS = ENV.fetch('KNOWN_HOSTS') { 'http://lvh.me:5000,http://localhost:4200,http://10.10.2.119:4200,http://0.0.0.0:4200' }
```

Comment:
This line should just be `'http://lvh.me:5000,http://localhost:4200'` at most, with the rest being set in `.env`. Please remove this change, and remove `.env` from source control.